pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "omf"
+version = "3.0.0-alpha"
+description = "API Library for Open Mining Format"
+authors = ["Global Mining Standards and Guidelines Group <it@seequent.com>"]
+repository = "http://github.com/GMSGDataExchange/omf"
+homepage = "http://www.globalminingstandards.org/"
+readme = "README.md"
+keywords = ["geology", "geophysics", "earth sciences"]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Programming Language :: Python',
+    'Topic :: Scientific/Engineering',
+    'Topic :: Scientific/Engineering :: Mathematics',
+    'Topic :: Scientific/Engineering :: Physics',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: Unix',
+    'Operating System :: MacOS',
+    'Natural Language :: English',
+]
+
+[tool.poetry.dependencies]
+python = "^3.7.2,<3.11"
+
+## pip dependencies
+geoh5py = {version = "0.4.0", source ="pypi"}
+numpy = {version = "^1.7", source = "pypi"}
+properties = {version = "0.4.0", source = "pypi"}
+pypng = {version = "^0.20220715", source = "pypi"}
+six = {version = "^1.16", source = "pypi"}
+vectormath = {version = "^0.2.0", source = "pypi"}
+setuptools = {version = "63.4.1", source = "pypi"}
+
+
+[tool.poetry.dev-dependencies]
+pylint = "^2.14.4"
+pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
+requests = "^2.27.1" # for tests only
```

Comment:
**question:** is `request` actually used in tests?

Reply:
Good point, removing